aspnetcore/fundamentals/servers/index.md

```diff
@@ -19,7 +19,12 @@ An ASP.NET Core app runs with an in-process HTTP server implementation. The serv
 
 ASP.NET Core ships two server implementations:
 
+::: moniker range="<= aspnetcore-2.0"
 * [Kestrel](xref:fundamentals/servers/kestrel) is a cross-platform HTTP server based on [libuv](https://github.com/libuv/libuv), a cross-platform asynchronous I/O library.
+::: moniker-end
+::: moniker range=">= aspnetcore-2.1"
+* [Kestrel](xref:fundamentals/servers/kestrel) is a cross-platform HTTP server based on managed sockets (`Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets`), a cross-platform asynchronous I/O library. In versions of ASP.NET Core prior to 2.1, Kestrel is based on [libuv](https://github.com/libuv/libuv).
```

Comment:
I also don't think discussion of the transport belongs here. If we must include it, I wouldn't call `Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets` a cross-platform asynchronous I/O library. That's what libuv is, but the sockets transport is more limited. It's just a socket library that implements Kestrel's transport abstraction.

Reply:
Sure, I can address that on the next commit.

Reply:
Yes, I think we should remove it. If I try to add that sentiment, it becomes perhaps a bit too complicated ...

> [Kestrel](xref:fundamentals/servers/kestrel) is a cross-platform HTTP server based on managed sockets (`Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets`), a cross-platform asynchronous I/O library. In versions of ASP.NET Core prior to 2.1, Kestrel is based on [libuv](https://github.com/libuv/libuv), which remains a viable alternative for 2.1 or later apps.

We can just go with this here ...

> > [Kestrel](xref:fundamentals/servers/kestrel) is a cross-platform HTTP server based on managed sockets (`Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets`), a cross-platform asynchronous I/O library.